doc/nrf/protocols/matter/getting_started/advanced_kconfigs.rst

```diff
@@ -129,12 +129,16 @@ To enable one of the reactions to the last fabric removal, set the corresponding
 * :kconfig:option:`CONFIG_CHIP_LAST_FABRIC_REMOVED_ERASE_AND_PAIRING_START` - Remove all saved network credentials and start Bluetooth LE advertising.
   The device will remove all saved network credentials, keep application-specific non-volatile data, and start advertising Bluetooth LE Matter service.
   After that, it will be ready for commissioning to Matter over Bluetooth LE.
+  This option is not available in the :ref:`matter_template_sample` sample.
```

Comment:
Actually why is that?

Reply:
After the f2f discussion, I've decided to add it to Matter Template to fix the build there.